src/Components/PressedKey.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+type State = {
+  key: string;
+};
+
+export class PressedKey extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    key: '',
```

Comment:
```suggestion
    pressedKey: '',
```
![image](https://github.com/mate-academy/react_keyboard/assets/96409563/3a120244-6943-4616-8d64-cb4caab16e0d)